Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,11 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * @return string
+     */
+    abstract protected function getContext();
```

Comment:
> but you added it... thats what the diff says here

Indeed lol … maybe you've worked on this PR for too long @EmmanuelVella ;) Please do rename it everywhere.

@Soullivaneuh : instead of commenting, the method could be made not abstract at first, then abstract on the master branch.